lib/internal/process/pre_execution.js

```diff
@@ -264,8 +264,10 @@ function setupFetch() {
   });
 
   // The WebAssembly Web API: https://webassembly.github.io/spec/web-api
-  const { wasmStreamingCallback } = require('internal/wasm_web_api');
-  internalBinding('wasm_web_api').setImplementation(wasmStreamingCallback);
+  const lazyWasmStreamingCallback = getLazy(() => require('internal/wasm_web_api').wasmStreamingCallback);
+  internalBinding('wasm_web_api').setImplementation((streamState, source) => {
+    lazyWasmStreamingCallback()(streamState, source);
```

Comment:
Seems like this `getLazy` is not necessary.

```suggestion
    require('internal/wasm_web_api').wasmStreamingCallback(streamState, source);
```

Reply:
That means we'd load `internal/wasm_web_api` unconditionally at startup. Moving the lazy call into the implementation callback means that we are only going to load the module when we need to stream wasm.

Reply:
Oh, I misread, the suggestion was still in the callback. In that case SGTM.